src/vanilla/DanModel/DanCodeGenerator.cs

```diff
@@ -5079,28 +5108,26 @@ internal static string MethodOperationExceptionTypeString(Method method, Setting
             return result;
         }
 
-        internal static IEnumerable<ParameterJv> MethodRetrofitParameters(Method method, Settings settings)
+        internal static IEnumerable<Parameter> MethodRetrofitParameters(Method method, Settings settings)
         {
-            IEnumerable<ParameterJv> result;
+            IEnumerable<Parameter> result;
 
             if (IsFluentOrAzure(settings))
             {
-                List<ParameterJv> parameters = MethodJvRetrofitParameters(method).ToList();
+                List<Parameter> parameters = MethodJvRetrofitParameters(method).ToList();
 
                 if (MethodIsPagingNextOperation(method))
                 {
                     parameters.RemoveAll(p => p.Location == ParameterLocation.Path);
 
-                    var nextUrlParam = new ParameterJv
-                    {
-                        Name = "nextUrl",
-                        SerializedName = "nextUrl",
-                        ModelType = DependencyInjection.New<PrimaryType>(KnownPrimaryType.String),
-                        Documentation = "The URL to get the next page of items.",
-                        Location = ParameterLocation.Path,
-                        IsRequired = true
-                    };
-                    nextUrlParam.Extensions.Add(AzureExtensions.SkipUrlEncodingExtension, true);
+                    Parameter nextUrlParam = DependencyInjection.New<Parameter>();
```

Comment:
What does DependencyInjection do now that we haven't specified any subtype of Parameter to inject? (or maybe I missed something)

Reply:
It just creates a Parameter object.